### PR TITLE
Use Github Actions for CI/CD instead of Gitlab

### PR DESCRIPTION
**Pull request by joonvena**
### Closes :no_good_woman:
**[AHTI-XXX](https://helsinkisolutionoffice.atlassian.net/browse/AHTI-XXX):**

### CODE REVIEW

File: .github/workflows/on_pull_request.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+name: Build & Deploy
```
Comment (ruriky) [inline .github/workflows/on_pull_request.yml:1]: I would suggest: `Build & Review`, `Build & Staging`  and `Build & Production` or.. `Build & Deploy Review`

File: .github/workflows/on_pull_request.yml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Build
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build
+        uses: andersinno/kolga-build-action@v2
+        with:
+          disable_cache: true
```
Comment (ruriky) [inline .github/workflows/on_pull_request.yml:29]: This should be default in the action as the problem with cache is ghcr related.

File: .github/workflows/on_pull_request.yml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    steps:
+      - uses: actions/checkout@v2
+      - uses: andersinno/kolga-setup-action@v2
+
+      - name: Review-Services
+        uses: andersinno/kolga-deploy-service-action@v2
+        with:
+          projects: CITY_OF_HELSINKI_AHTI
+        env:
+          ENVIRONMENT_URL: https://${{ env.K8S_NAMESPACE }}.newtest.kuva.hel.ninja
```
Comment (ruriky) [inline .github/workflows/on_pull_request.yml:49]: Should we put the domain `newtest.kuva.hel.ninja` in a global variable or even better.. manage it with TF.

File: .github/workflows/on_pull_request.yml [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+          POSTGRES_IMAGE: "docker.io/andersinnovations/postgis:11-bitnami"
+
+      - name: Deploy
+        uses: andersinno/kolga-deploy-action@v2
+        env:
```
Comment (ruriky) [inline .github/workflows/on_pull_request.yml:54]: Application specific variables could be groupped and introduced after the mandatory ones (like ENVIRONMENT_URL)

**Review by ruriky — CHANGES_REQUESTED** [review]

File: .github/workflows/on_pull_request.yml (hunk 3 above)
Comment (joonvena) [reply .github/workflows/on_pull_request.yml:49]: We should

**Review by ruriky — APPROVED** [review]
LGTM

**Pull request merged**